packages/styled-react/src/__tests__/__snapshots__/exports.test.ts.snap

```diff
@@ -2,6 +2,7 @@
 
 exports[`@primer/styled-react exports 1`] = `
 [
+  "ToggleSwitch",
```

Comment:
looks like this was previously following alphabetical order, can we fix back ?

Reply:
`exports.test.ts.snap` is auto-generated by the test runner.
I guess the reason `ToggleSwitch` appears first is that it's handled differently from other components, it's wrapped with `createStyledComponent` and exported separately.